@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1350,9 +1344,6 @@ export class TextElementsRenderer {
 
         const placeNew = this.m_forceNewLabelsPass || placeNewTextElements;
         if (this.m_forceNewLabelsPass) {
-            if (!placeNewTextElements) {
-                logger.debug("Force new label pass");
-            }
```

Comment:
This log indicates when a pass for new labels is done because there were glyphs still loading, or labels are elevated on terrain and terrain data was not yet loaded. It's useful to debug issues related to that and to know why the text renderer keeps doing text placement even if there's no new labels added.

Reply:
I don't think this is a good reason.